server/game/cards/15-W2D/TheAngler.js

```diff
@@ -0,0 +1,35 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class TheAngler extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Call Out',
+            playType: 'noon',
+            target: {
+                location: 'play area',
+                cardCondition: card => card.locationCard === this.locationCard && card.value === this.value && card.getType() === 'dude' && card.controller !== this.controller
+            },
+            message: context => {
+                this.game.addMessage('{0} uses {1} to call out {2}', this.controller, this, context.target);
+            },
+            handler: context => {
+                this.game.resolveGameAction(GameActions.callOut({ caller: this, callee: context.target }), context);
+            }
+        });
+
+        this.action({
+            title: 'Change Value',
+            playType: 'noon',
+            cost: ability.costs.discardFromHand(),
+            handler: context => {
+                this.value === context.costs.discardFromHand.value;
```

Comment:
This would change the value permanently, but this is just a noon ability. You need to use effect to set value.
```suggestion
                this.applyAbilityEffect(context.ability, ability => ({
                    match: this,
                    effect: ability.effects.setValue(context.costs.discardFromHand.value)
                }));                
```